src/Stage.js

```diff
@@ -55,60 +56,72 @@ export const includingCanvasProps = key => !includingDisplayObjectProps(key) &&
 export const getCanvasProps = props => filterByKey(props, includingCanvasProps);
 export const getDisplayObjectProps = props => filterByKey(props, includingDisplayObjectProps);
 
-class Stage extends React.Component {
-  componentDidMount() {
-    const { children, height, options, width } = this.props;
+export let appTestHook = null;
```

Comment:
We can no longer rely on the instance property `_app` on Stage for tests now that it has been refactored into a function component. This is a temporary solution to expose the most recently created pixi application instance so tests can hook into Stage internals.